packages/espree/tsconfig-cjs.json

```
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "lib": ["ESNext"],
+    "moduleResolution": "node",
+    "module": "commonjs",
+    "allowJs": true,
+    "esModuleInterop": true,
+    "noEmit": false,
+    "declaration": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": true,
+    "strict": true,
```

```suggestion
    "strict": true,
    "exactOptionalPropertyTypes": true,
```

Same as the comment above!